Networking/Networking/Model/SitePlugin.swift

```diff
@@ -0,0 +1,156 @@
+import Foundation
+
+/// Represents a specific plugin entity for a specific site.
+///
+public struct SitePlugin: Decodable, GeneratedFakeable {
+    public let siteID: Int64
+    public let plugin: String                   // e.g. woocommerce/woocommerce (i.e. [folder/]main php file)
+    public let status: String                   // e.g. active, inactive, ...
```

Comment:
nit: could be better to use an `enum` here if we already know all the statuses

Reply:
Agreed. There are three: active, inactive and network-active. I'll refactor.

https://github.com/WordPress/WordPress/blob/35e1b34f8ee4fcfc8f5e12d48f665b8d599c6e89/wp-includes/rest-api/endpoints/class-wp-rest-plugins-controller.php#L61

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3844/commits/8e320fa88e27122e36d57a3de6e396bc6ec0ce37